src/RouteResolver.tsx

```diff
@@ -116,7 +117,13 @@ const RouteResolver: React.FunctionComponent<RouteComponentProps> = ({
   if (document.title !== newTitle) {
     document.title = newTitle
   }
-
+  const scrollToRef = React.useRef<HTMLElement>(null)
```

Comment:
If the hash value (from the URL) is set, and it matches the title of one of the Synapse components on the page, then it will scroll to it (with an offset to respect the top nav menu height).